src/components/bottom-bar/bottom-bar.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+
+import styles from './bottom-bar-styles.scss';
+
+/**
+ * Bottom Bar component
+ */
+const BottomBar = ({ theme, linkRoute, linkTitle, footerText }) => (
+  <div className={cx(styles.container, theme.container)}>
+    <div
+      className={cx(styles.row, styles.bottomBar, theme.row, theme.bottomBar)}
+    >
+      <Link className={styles.text} to={linkRoute}>
```

Comment:
So instead of using this library dependency component allow passing a `children` where every project will use its own links.